.github/workflows/codeql.yml

```diff
@@ -0,0 +1,60 @@
+# Define the name of the workflow
+name: "CodeQL-Advanced"
+
+# Define when the workflow should be triggered (on push to a specific branch and pull requests to the master branch)
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
```

Comment:
There is no `"master"` branch, use `"main"`

Reply:
Updated the branch details